app/src/components/main/navigationTabs/navigationTabs.scss

```diff
@@ -27,7 +27,7 @@
   box-sizing: border-box;
   width: 100%;
   padding-top: 15px;
-  border-bottom: 1px solid $COLOR--gray-91;
+  border-bottom: 2px solid $COLOR--gray-91;
```

Comment:
I am not sure, that we need to change the width of this line.
https://www.figma.com/file/PxNk9h6CS9Y4Mxcl2fS9HF/RP5-Project-Level?type=design&node-id=1453-144839&mode=design&t=MVVjCbM1Y6Y6nktj-0
The same component with 1px border width
Discuss this with BA and UI/UX

Reply:
according to the design from the ticket:
https://www.figma.com/file/PxNk9h6CS9Y4Mxcl2fS9HF/RP5-Project-Level?type=design&node-id=7756-47207&mode=design&t=UU8oBhhWIdmc6L2H-0